### ISSUE TITLE
Error messages take keyboard focus

### ISSUE THREAD

**Issue by clegginabox** [benmatselby/sublime-phpcs#104]
Apologies if this has already been mentioned, I had a quick scan through previous issues but didn't find anything.

This may just be Sublime functionality but is it possible for the error messages not to take the keyboard focus? Sometimes I will accidentally hit save before typing a closing bracket, the error messages will appear and take my keyboard focus away from where the closing bracket should be. It's only a case of having to hit Esc to return focus but it happens often enough that it's quite annoying.
